api/spec/src/billing/invoices/invoice.tsp

```diff
@@ -240,27 +267,57 @@ enum LineTypes {
 @discriminator("type")
 @friendlyName("BillingLine")
 model Line {
+  ...OmitProperties<global.Resource, "id">;
+
   @visibility("read", "query", "update")
   id: ULID;
 
   type: LineTypes;
 
+  @visibility("read", "query")
+  status: LineStatus;
+
   @summary("Discounts applied to this line.")
   discounts?: LineDiscount[];
 
   @summary("Charges applied to this line.")
   charges?: LineCharge[];
 
+  @summary("The invoice this item belongs to.")
+  invoice?: InvoiceReference;
+
+  @summary("The currency of this line")
+  currency: CurrencyCode;
+
   @summary("Map of taxes to be applied and used in the invoice totals.")
   @visibility("read", "query") // Tax is handled by the tax provider, we only need to show the output of that here
   taxes?: TaxItem[];
 
+  @summary("Tax overrides specify the tax configuration for this line.")
+  taxOverrides?: OpenMeter.ProductCatalog.TaxConfig | null;
+
   @visibility("read")
   @summary("Total sum of the line, including discounts and charges.")
   total: Numeric;
 
   @summary("Period of the line item applies to for revenue recognition pruposes.")
   period: Period;
+
+  @summary("The time this line item should be invoiced.")
+  invoiceAt: DateTime;
+
+  @summary("Quantity of the item being sold.")
+  quantity?: Numeric;
```

Comment:
shouldn't this default to one? Does it make sense selling something without quantity?

Reply:
It's not mandated due to the ubp story. We need qty but in that case it doesn't need to be set during creation.